data/mods/Magiclysm/Spells/monsterspells.json

```diff
@@ -16,5 +16,44 @@
     "range_increment": 0.2,
     "effect": "projectile_attack",
     "extra_effects": [ { "id": "light_healing", "hit_self": true } ]
+  },
+  {
+    "id": "rocket_punch",
+    "type": "SPELL",
+    "name": "Rocket Punch",
+    "description": "Ejects giant fist from arm.",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "NO_LEGS" ],
+    "damage_type": "bash",
+    "min_damage": 30,
+    "max_damage": 40,
+    "damage_increment": 2,
+    "max_level": 5,
+    "min_range": 1,
+    "max_range": 2.5,
```

Comment:
this value is incorrect as max_range is an integer. 2.5 will be truncated to 2, so you either need to make it 3 or 2.